app/routes/apply/business.js

```diff
@@ -0,0 +1,21 @@
+const getBusinessViewModel = require('../../view-models/apply/business')
+const landService = require('../../services/land-service')
+
+module.exports = {
+  method: 'GET',
+  path: '/apply/{sbi}',
+  options: {
+    handler: async (request, h) => {
+      try {
+        const { sbi } = request.params
```

Comment:
SBI is passed through to the page, as well as the api call with no validation other than the default route validation built into Hapi. I couldn't manage to exploit this, but I think the input should be validated by using the validate option in the route config.

Reply:
You're quite right. I've added validation for SBIs so they must be nine characters, all numeric.